tree-sitter-python/tree-sitter-python.cabal

```diff
@@ -23,6 +23,7 @@ library
                      , filepath         ^>= 1.4
                      , template-haskell  >= 2.12.0.0 && < 2.15.0.0
                      , tree-sitter-python-internal
+                     , semantic-source   >= 0.0.0.0
```

Comment:
You probably want ^>= and not >= (since that would match all future versions of semantic-source).

Reply:
Ohhh I got it the other way around. Thx!